tests/onnx/test_e2e_ptq.py

```diff
@@ -330,32 +346,96 @@ def get_ac_command(task_type: str, model_name: str, model_dir: Path,
 
         return com_line
 
+    @staticmethod
+    def get_ov_ac_command(task_type: str, model_name: str, model_dir: Path,
+                          data_dir: Path, anno_dir: Path, output_dir: Path,
+                          eval_size: int, program: str, is_quantized: bool) -> List[str]:
+
+        program_path = BENCHMARKING_DIR / program
+
+        task_path = BENCHMARKING_DIR / task_type
+        config_path = task_path / "openvino_models_configs" / (model_name + ".yml")
+
+        output_dir = output_dir / task_type
+        if not output_dir.exists():
+            output_dir.mkdir(parents=True)
+
+        anno_dir = anno_dir / str(eval_size)
+        if not anno_dir.exists():
+            anno_dir.mkdir(parents=True)
+
+        out_file_name = os.path.splitext(program)[0]
+
+        if is_quantized:
+            out_file_name += "-quantized.csv"
+        else:
+            out_file_name += "-reference.csv"
+
+        model_file_name = model_name + "-quantized" if is_quantized else model_name
+        model_file_name += ".onnx"
+
+        com_line = [
+            sys.executable, str(program_path),
+            "-c", str(config_path),
+            "-d", str(DATASET_DEFINITIONS_PATH_ONNX),
+            "-m", str(model_dir / task_type / model_file_name),
+            "-s", str(data_dir),
+            "-a", str(anno_dir),
+            "--csv_result", str(output_dir / out_file_name)
+        ]
+        if eval_size is not None:
+            com_line += ["-ss", str(eval_size)]
+
+        return com_line
```

Comment:
You duplicate too much code here. Refactor to avoid duplication.

Reply:
Done